src/database.ts

```diff
@@ -1336,7 +1336,7 @@ class Database extends GrpcServiceObject {
         reqOpts,
         gaxOpts,
       },
-      (err, sessions, ...args) => {
+      (err, sessions, nexPageRequest, ...args) => {
```

Comment:
What is the format/properties of `nextPageRequest`?

Reply:
`nextPageRequest` comes in as `proto.~.IList<resource>Request` which is usually
```ts
parent/database;
pageSize;
pageToken;
//some request objects also have 
filter
```